recipes/emcee/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.1.0" %}
+
+package:
+  name: emcee
+  version: {{ version }}
+
+source:
+  fn: emcee-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/de/00/2358f12c98fa74ab58d78e6a4a4f9a8152fadf6ade2d809d98d771dcc31b/emcee-{{ version }}.tar.gz
```

Comment:
Can you change that to https://pypi.io/packages/source/e/emcee/emcee-2.1.0.tar.gz ?